static/js/voting_widget.js

```diff
@@ -77,11 +77,7 @@ exports.poll_data_holder = function (is_my_poll, question) {
                 };
                 poll_question = question;
 
-                if (is_my_poll) {
                     return event;
-                }
-                return;
```

Comment:
This is supposed to be a feature that only the user who created the poll can add/edit the question. Hence, here we don't return the `question` event for the users who didn't create the poll. This change would remove that feature and allow any user to add/edit the question.

I added this feature after having a discussion with @showell. Maybe we can leave a comment here stating the reason why we are doing this, just to make it clear.

Reply:
Okay, but this is already enforced in the frontend in `render_results`:
```js
        if (!is_my_poll) {
            if (widget_data.question !== '') {
                // For the non-senders, we hide the question input bar
                // when we have a question assigned to the poll
                elem.find('.poll-question-bar').hide();
            } else {
                // For the non-senders we disable the question input bar
                // when we have no question assigned to the poll
                elem.find('button.poll-question').attr('disabled', true);
                elem.find('input.poll-question').attr('disabled', true);
            }
        }
```
anyway it's a good idea for having that, so I'll remove this commit if this doesn't makes sense.